test/Infer/precision-kb-2.opt

```diff
@@ -0,0 +1,11 @@
+; REQUIRES: solver
+; RUN: %souper-check %solver -infer-rhs -infer-known-bits %s
+; RUN: %souper2llvm %s | %llvm-as | %souper %solver -print-known-at-return
+
+; CHECK: knownBits from souper: xxxxxxx0
+; CHECK: knownBits from compiler: xxxxxxxx
```

Comment:
it's great to test that souper returns the precise result, but do not test that the compiler returns the imprecise result, that is not something that we want our tests relying upon

Reply:
I added this check to only test what LLVM returns, but not with an intent to compare the precision of Souper vs LLVM. But, I will take off this LLVM check part from here and add relevant tests for LLVM where we can expect some useful dataflow information.